keras_core/backend/numpy/math.py

```diff
@@ -1,7 +1,10 @@
 import numpy as np
+import scipy.signal
```

Comment:
When using the numpy backend, scipy might not be installed. Prefer using `keras.utils.module_utils` to create a lazily imported package.

Reply:
Added